modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,132 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: PROCEDURE
+[id="logging-loki-deploy_{context}"]
+= Deploying the LokiStack
+You can use the {product-title} web console to install the LokiOperator.
+
+.Prerequisites
+
+* {logging-title-uc} Operator 5.5
+* S3 bucket for log storage
+
+To install the LokiStack using the {product-title} web console:
+
+. Install the LokiOperator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *LokiOperator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces on the cluster*.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and could publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+
+.. Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+You must select this option to ensure that cluster monitoring
+scrapes the `openshift-operators-redhat` namespace.
+
+.. Select an *Approval Strategy*.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.. Verify that you installed the LokiOperator. Visit the *Operators* → *Installed Operators* page and look for "LokiOperator."
+
+.. Ensure that *LokiOperator* is listed with *Status* as *Succeeded* in all the projects.
++
+. Create a Secret YAML file that uses the aws_access_key_id and aws_secret_access_key fields to specify your base64-encoded AWS credentials. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+----
++
+. Create `LokiStack` Custom Resource
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: logging-loki
+    namespace: openshift-logging
+  spec:
+    size: 1x.small
+    storage:
+      schemas:
+      - version: v12
+        effectiveDate: 2022-06-01
+      secret:
+        name: logging-loki-s3
+        type: s3
+    storageClassName: gp2
+    tenants:
+      mode: openshift-logging
+    rules:
+      enabled: true
+      selector:
+        matchLabels:
+          openshift.io/cluster-monitoring: "true"
+      namespaceSelector:
+        matchLabels:
+          openshift.io/cluster-monitoring: "true"
```

Comment:
Can you mark the ruler component as "optional"? It's only needed if someone wants to try out the log-based metrics / alerts (`RecordingRule` / `AlertingRule` CRDs, which are still tech-preview).

Another option would be to leave the ruler out of the basic configuration completely, if only GA features should make it to this howto document.

Reply:
Thank you for the feedback, I'm going to remove the rules section in lieu of notating that it's tech-preview only for the sake of clarity.